sample/src/main/java/com/rbrooks/indefinitepagerindicatorsample/recyclerViewSample/RecyclerViewSampleFragment.kt

```diff
@@ -33,7 +33,7 @@ class RecyclerViewSampleFragment : Fragment(), OnPagerNumberChangeListener {
     }
 
     private fun setupViews() {
-        recyclerViewAdapter = PhotoItemRecyclerAdapter(context)
+        recyclerViewAdapter = PhotoItemRecyclerAdapter(context!!)
```

Comment:
`!!`, this is not a safe call, context can be null (although it is very unlikely). It would probably be good to add a check for null context in PhotoItemRecyclerAdapter.